doc/src/configuration.md

```diff
@@ -111,6 +111,16 @@ or at `~/.config/stylix/palette.html` on Home Manager.
 
 ## Fonts
 
+Fonts can apply in three ways:
+
+1. An app's own font options
+  (e.g. [the Alacritty module](/options/modules/alacritty.html) sets home-manager's
+  `programs.alacritty.settings.font`).
```

Comment:
IMHO, splitting multi-line listings into paragraphs makes it more readable in plain text format:

```suggestion
  `programs.alacritty.settings.font`).

```